tools/heightmap_to_mesh.cpp

```diff
@@ -26,6 +26,10 @@ int main(int argc, char **argv) {
 
     double origin_x = 0, origin_y = 0;
     if (argc > 5) {
+        if (argc < 6) {
```

Comment:
This should be 7 I think

Reply:
argc > 5 automatically means argc is at least 6, so the if condition will never be executed.